### PR TITLE
refactor: EN-509 only including files used for publishing

### PR DESCRIPTION
**Pull request by James-FarmerJoe**
James Cullimore has to register every part of his body as a separate lethal weapon. His spleen is considered a concealed weapon in over 50 states.

### CODE REVIEW

**Comment by James-FarmerJoe** [comment]
To ensure this works I will need to make a release then pull the 2 change repos (mobile & functions) and test them

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@farmerjoeorg/farmerjoe-language",
-  "version": "0.4.7",
+  "version": "0.5.0",
```
Comment (philipheinser) [inline package.json:3]: should this be 1.0.0 since its a breaking change?

**Review by philipheinser — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (James-FarmerJoe) [reply package.json:3]: Yeah I wasnt too sure if it should be minor or major change as we have introduced code breaking chages in some repos yet only bumped the minor. Will look into your other comments to see if further changes are needed then I'll change it to 1.0.0 😄

File: package.json [hunk 2]
```diff
@@ -30,13 +30,10 @@
   "main": "dist/index.js",
   "files": [
     "dist/",
-    ".flowconfig",
     "CountryUtil.js",
     "callCodes.json",
-    "crowdin.yml",
     "generateCallCodes.js",
     "index.js",
```
[the remark this answers is not in the document]
Comment (James-FarmerJoe) [reply package.json:36]: I couldnt see them in the dist folder and will need to look into where they are even used before removing them

**Review by vpenkoff — APPROVED** [review]

**Review by fj-vp — APPROVED** [review]

**Review by mahyarfarmerjoe — APPROVED** [review]

**Pull request merged**